.github/workflows/r.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: R
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run: 
+        working-directory: Syndemics
+    strategy:
+      matrix:
+        r-version: ['4.1.1']
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up R ${{ matrix.r-version }}
+        uses: r-lib/actions/setup-r@f57f1301a053485946083d7a45022b278929a78a
```

Comment:
Is there something about the version of the action at this specific commit hash that you need? I'm not 100% sure what's failing yet. In this step of the workflow, though, the output is
```
Reading package lists...
E: The repository 'https://ppa.launchpadcontent.net/cran/travis/ubuntu jammy Release' does not have a Release file.
Repository: 'deb https://ppa.launchpadcontent.net/cran/travis/ubuntu/ jammy main'
Description:
Backports of important libraries used by R packages on Travis
More info: https://launchpad.net/~cran/+archive/ubuntu/travis
Adding repository.
Adding deb entry to /etc/apt/sources.list.d/cran-ubuntu-travis-jammy.list
Adding disabled deb-src entry to /etc/apt/sources.list.d/cran-ubuntu-travis-jammy.list
Adding key to /etc/apt/trusted.gpg.d/cran-ubuntu-travis.gpg with fingerprint 4A0C193118803EB4A561E569B3CF35C315B55A9F
/usr/bin/sudo DEBIAN_FRONTEND=noninteractive apt-get update -y -qq
E: The repository 'https://ppa.launchpadcontent.net/cran/travis/ubuntu jammy Release' does not have a Release file.
Error: Failed to get R 4.1.1: Failed to get R 4.1.1: Failed to install R: Error: The process '/usr/bin/sudo' failed with exit code 100
```
which suggests that it's failing to install an unofficial Ubuntu package. Unless they changed very recently, `ubuntu-latest` should be using Ubuntu 22.04, which ships `R version 4.1.2`, installable via 
```sh
sudo apt install r-base
```
which may work the same for your needs?

Reply:
hmm - this was a portion that I changed and asked it to run on linux as macos (the default) costs more, although seeing this is a public repo I believe it's free regardless

Reply:
switch to macos still seems like it failed - although I think it's currently failing because it's waiting for the patch from #8